src/System.CommandLine/Command.cs

```diff
@@ -57,28 +57,31 @@ public IEnumerable<Symbol> Children
         }
 
         /// <summary>
-        /// Represents all of the arguments for the command.
+        /// Gets all of the arguments for the command.
         /// </summary>
         public IList<Argument> Arguments => _arguments ??= new(this);
 
         internal bool HasArguments => _arguments?.Count > 0 ;
 
         /// <summary>
-        /// Represents all of the options for the command, inherited options that have been applied to any of the command's ancestors.
+        /// Gets all of the options for the command.
         /// </summary>
+        /// <remarks>
+        /// This collection doesn't include options on parent commands where <see langword="Option.Recursive" /> is <see langword="true" />. Those options are valid under the current command but don't appear in this collection.
```

Comment:
`langword="Option.Recursive"` should be `cref`.

And I somewhat expect that this mistake should have been detected by some test in CI.